src/material/form-field/_form-field-fill-theme.scss

```diff
@@ -9,19 +8,33 @@
 @mixin mat-form-field-fill-theme($theme) {
   $foreground: map-get($theme, foreground);
   $is-dark-theme: map-get($theme, is-dark);
-
-  $fill-background: mat-color($foreground, base, if($is-dark-theme, 0.1, 0.04));
+  $fill-background: mat-color($foreground, base);
   $fill-disabled-background: mat-color($foreground, base, if($is-dark-theme, 0.05, 0.02));
+  $ripple-hover-height: 1px !default;
   $underline-color: mat-color($foreground, divider, if($is-dark-theme, 0.5, 0.42));
   $label-disabled-color: mat-color($foreground, disabled-text);
+  // The border radius for the form field fill box.
+  $mat-form-field-fill-border-radius: 4px !default;
 
   .mat-form-field-appearance-fill {
-    .mat-form-field-flex {
-      background-color: $fill-background;
+    .mat-form-field-flex::before {
+      background: $fill-background;
+      content: '';
```

Comment:
some of these styles don't have to do with theme, they should go in `form-field-fill.scss`